.github/workflows/sanity.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     strategy:
```

Comment:
Maybe we should start running on nightly and wait for stability before running all the time on sanity ? 
WDYT ?

Reply:
I think sanity, it's the first line...

It run on each PR. That allow us to be vigilant to be stable.

(And honestly, no problem on the validation on each PR).